scripts/install/install-pocs.sh

```diff
@@ -255,7 +269,7 @@ function get_or_build_images {
     if ${DEVELOPER}; then
         echo "Building local PANOPTES docker images."
 
-        cd "${POCS}"
+        cd "${PANDIR}/POCS"
         ./docker/setup-local-environment.sh
```

Comment:
Returned `Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post http://%2Fvar%2Frun%2Fdocker.sock/v1.40/containers/prune: dial unix /var/run/docker.sock: connect: permission denied` when testing the script in a VM.

I fixed this by running `sudo chmod 666 /var/run/docker.sock`.

This may just be a VM issue.

Reply:
Was that after the full install or during the docker pull?

Reply:
This was during the docker pull.

Reply:
When the `setup-local-environment` script ran the `docker system prune --force` command.